Update investigations (#834)
opensciencegrid/technology: docs/meetings/2020/TechArea20201005.md

### Support Update

-   Univeristy of Connecticut (BrianL): investigating Blahp segfault with Flightworthy

> -   Lots of support for jlab + origin.  Host is crashing.  Not necessarily the OSG provided container.

Reviewer: ```suggestion
-   JLab (Dereek): Origin host is crashing.  Not necessarily the OSG provided container.
```
